docs/source/contributing/tests.rst

```diff
@@ -11,10 +11,9 @@ can find them under the `jupyterhub/tests <https://github.com/jupyterhub/jupyter
 
 Running the tests
 ==================
-
-#. Make sure you have completed :ref:`contributing/setup`. Once completed, you should be able
-   to run ``jupyterhub`` on your command line and access JupyterHub from your browser at http://localhost:8000. Being able to run and access `jupyterhub` should mean that the dev environment is properly set
-   up for tests to run.
+ #Make sure you have completed :ref:`contributing/setup`. Once you are done,you would be able
```

Comment:
This loses the number-list formatting. Make sure the line still starts with `#. `. Missing space after comma.